Makefile

```diff
@@ -145,6 +145,11 @@ build-installer: manifests generate kustomize ## Generate a consolidated YAML wi
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default > dist/install.yaml
 
+# Short version of hack/package-helm-charts.sh
+.PHONY: helm-charts
+helm-charts:
```

Comment:
@ArangoGutierrez lets add all CI changes required for Helm packaging together in one MR. In this case its not clear which image will be used with this target as that will not match with `VERSION`.

Reply:
sounds good.